routes/post.go

```diff
@@ -958,6 +963,129 @@ func (fes *APIServer) GetPostsStateless(ww http.ResponseWriter, req *http.Reques
 	}
 }
 
+// GetPostsHashlistRequest ...
+type GetPostsHashlistRequest struct {
+	PostsHashlist              []string `safeForLogging:"true"`
+	ReaderPublicKeyBase58Check string `safeForLogging:"true"`
+	OrderBy                    string `safeForLogging:"true"`
+}
+
+// GetPostsHashlistResponse ...
+type GetPostsHashlistResponse struct {
+	PostsFound []*PostEntryResponse
+	PostsSkipped []*SkippedPostEntryResponse
+}
+
+func (fes *APIServer) GetPostsHashlist(ww http.ResponseWriter, req *http.Request) {
+	decoder := json.NewDecoder(io.LimitReader(req.Body, MaxRequestBodySizeBytes))
+	requestData := GetPostsHashlistRequest{}
+	if err := decoder.Decode(&requestData); err != nil {
+		_AddBadRequestError(ww, fmt.Sprintf("GetPostsHashlist: Problem parsing request body: %v", err))
+		return
+	}
+
+	// Decode the reader public key into bytes. Default to nil if no pub key is passed in.
+	var readerPublicKeyBytes []byte
+	if requestData.ReaderPublicKeyBase58Check != "" {
+		var err error
+		readerPublicKeyBytes, _, err = lib.Base58CheckDecode(requestData.ReaderPublicKeyBase58Check)
+		if requestData.ReaderPublicKeyBase58Check != "" && err != nil {
+			_AddBadRequestError(ww,
+				fmt.Sprintf("GetPostsHashlist: Problem decoding user public key: %v : %s", err, requestData.ReaderPublicKeyBase58Check))
```

Comment:
```suggestion
				fmt.Sprintf("GetPostsHashlist: Problem decoding reader public key: %v : %s", err, requestData.ReaderPublicKeyBase58Check))
```

Reply:
Fixed in mine, but found in more functions